libraries/AP_BLHeli/AP_BLHeli.cpp

```diff
@@ -151,12 +151,23 @@ const AP_Param::GroupInfo AP_BLHeli::var_info[] = {
     // @RebootRequired: True
     AP_GROUPINFO("RVMASK",  12, AP_BLHeli, channel_reversed_mask, 0),
 
+#if HAL_WITH_ESC_TELEM
```

Comment:
I think there should be a separate define for this since it is not generically supported by BLHeli ESCs

Reply:
This PR now has changes related to DroneCAN ESCX implementation only. BLHeli implementation is removed from this PR.